appf.adoc

```diff
@@ -79,17 +79,30 @@ __Map parameters:__::
 * **`sweep_angle_axis`**
 * **`fixed_angle_axis`**
 
-__Map coordinates:__:: The x (abscissa) and y (ordinate) rectangular coordinates are identified by the **`standard_name`** attribute values **`projection_x_coordinate`** and **`projection_y_coordinate`** respectively. In the case of this projection, the projection coordinates in this projection are directly related to the scanning angle of the satellite instrument, and their units are radians.
+__Map coordinates:__:: The x (abscissa) and y (ordinate) rectangular coordinates are identified by the **`standard_name`** attribute values **`projection_x_angular_coordinate`** or **`projection_x_coordinate`** (deprecated) and **`projection_y_angular_coordinate`** or **`projection_x_coordinate`** (deprecated), respectively.
```

Comment:
This sentence has a lot of "or"s and "and"s. Perhaps an extra comma after the first "(deprecated)" might help parsing?

__Map coordinates:__:: The x (abscissa) and y (ordinate) rectangular coordinates are identified by the **`standard_name`** attribute values **`projection_x_angular_coordinate`** or **`projection_x_coordinate`** (deprecated), **[[new comma here]]** and **`projection_y_angular_coordinate`** or **`projection_x_coordinate`** (deprecated), respectively.

Reply:
How about a more direct approach? I think it will be less confusing.

__Map coordinates:__:: The x (abscissa) and y (ordinate) rectangular coordinates are identified by the **`standard_name`** attribute values **`projection_x_angular_coordinate`** and **`projection_y_angular_coordinate`** respectively. CF specified the standard names **`projection_x_coordinate`** and  **`projection_y_coordinate`** for these coordinates prior to version 1.9, but that use is deprecated.

Reply:
That's much better! Thanks

Reply:
Implemented in c73544e